workitem/link/link_repository_blackbox_test.go

```diff
@@ -892,3 +892,11 @@ func (s *linkRepoBlackBoxTest) TestDeleteLinkAndListChildren() {
 		require.Len(t, childrenList, 0)
 	})
 }
+
+func (s *linkRepoBlackBoxTest) TestDeleteLink() {
+	s.T().Run("ok", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB, tf.WorkItemLinks(1))
+		err := s.workitemLinkRepo.DeleteRelatedLinks(s.Ctx, fxt.WorkItems[0].ID, fxt.Identities[0].ID)
+		require.NoError(t, err)
```

Comment:
Is it possible to add more assertion here? Other than just checking error, if it is possible to assert that there is no link exists, that would be better.

Reply:
addressed here 3d6aabb